src/main/java/bio/terra/service/configuration/StatusService.java

```diff
@@ -0,0 +1,102 @@
+package bio.terra.service.configuration;
+
+import bio.terra.model.RepositoryStatusModel;
+import bio.terra.model.RepositoryStatusModelSystems;
+import bio.terra.service.dataset.DatasetDao;
+import bio.terra.service.iam.IamProviderInterface;
+import org.broadinstitute.dsde.workbench.client.sam.ApiException;
+import org.broadinstitute.dsde.workbench.client.sam.model.SystemStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataAccessException;
+import org.springframework.stereotype.Service;
+
+import java.util.concurrent.Callable;
+
+@Service
+public class StatusService {
+
+    private final Logger logger = LoggerFactory.getLogger(StatusService.class);
```

Comment:
logger can be `static`
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(StatusService.class);
```